data/transport missions.txt

```diff
@@ -1975,3 +1975,44 @@ ship "Science Drone"
 		"heat dissipation" 1
 	explode "tiny explosion" 5
 	explode "small explosion" 2
+
+
+
+mission "Loan Broker Reunion"
+	name "Transfer enforcers to <planet>"
+	description "Transport 15 enforcers from <source> to <destination> by <day> to fulfill a request from the New Boston loan broker."
+	deadline
+	passengers 15
+	minor
+	to offer
+		random < 25
+		unpaid mortages = 0
```

Comment:
This should be `==`.

Reply:
I also think `unpaid mortgages` (not `mortages`) should be covered by quotation marks.